source/reference/api/progress.txt

```diff
@@ -15,7 +15,7 @@
 Description
 -----------
 
-Returns the status of the synchronization process.
+Returns either an updated status of the synchronization process or an error.
```

Comment:
I think this typo still needs to be fixed on this page:

![Screenshot 2024-01-11 at 1 17 05 PM](https://github.com/mongodb/docs-cluster-to-cluster-sync/assets/42851913/13103a2b-ebf6-4064-abd3-ca2f95b3e28c)

Reply:
Fixed